setup.cfg

```diff
@@ -1,3 +1,20 @@
 [flake8]
 exclude = .git,venv,env
 max-line-length = 79
+
+[semantic_release]
+; publish every time you push to 'dev'
+branch = dev
+; multiple version variables can be defined, comma-separated
+version_variable = foca/__init.py__:__version__
```

Comment:
So will this auto-upgrade all defined `version_variable` instances or is this to actually _get_ the version we want to bump? Does this mean we still need to manually bump the version? What is the commit parser good for then? I should probably read this myself, but guess it's just as well to leave the question here so that I remember it tomorrow.

Reply:
The former, it gets auto-upgraded every time the commit parser detects an upgrade. It's just a explicitly defined place in the code where the version "lives".